main_test.go

```diff
@@ -240,59 +240,91 @@ var _ = Describe("github-review-helper", func() {
 				})
 
 				Context("with a +1 comment", func() {
-					BeforeEach(func() {
-						requestJSON = `{
-  "issue": {
-    "number": 7,
-    "pull_request": {
-      "url": "https://api.github.com/repos/salemove/github-review-helper/pulls/7"
-    }
-  },
-  "comment": {
-    "body": "+1, awesome job!"
-  },
-  "repository": {
-    "name": "github-review-helper",
-    "owner": {
-      "login": "salemove"
-    },
-    "ssh_url": "git@github.com:salemove/github-review-helper.git"
-  }
-}`
-						mockSignature()
-					})
+					var itMarksCommitPeerReviewed = func() {
+						Context("with GitHub request failing", func() {
+							BeforeEach(func() {
+								pullRequests.On("Get", "salemove", "github-review-helper", 7).Return(nil, nil, errors.New("an error"))
+							})
 
-					Context("with GitHub request failing", func() {
-						BeforeEach(func() {
-							pullRequests.On("Get", "salemove", "github-review-helper", 7).Return(nil, nil, errors.New("an error"))
+							It("fails with a gateway error", func() {
+								handle()
+								Expect(responseRecorder.Code).To(Equal(http.StatusBadGateway))
+							})
 						})
 
-						It("fails with a gateway error", func() {
-							handle()
-							Expect(responseRecorder.Code).To(Equal(http.StatusBadGateway))
+						Context("with GitHub request succeeding", func() {
+							BeforeEach(func() {
+								pullRequests.On("Get", "salemove", "github-review-helper", 7).Return(&github.PullRequest{
+									Head: &github.PullRequestBranch{
+										SHA: github.String("1235"),
+										Ref: github.String("feature"),
+									},
+								}, nil, nil)
+							})
+
+							It("reports the status and succeeds", func() {
+								repositories.On("CreateStatus", "salemove", "github-review-helper", "1235", mock.AnythingOfType("*github.RepoStatus")).Return(nil, nil, nil)
```

Comment:
As for the line length, I've started moving the calls to multiple lines in other commits. I'll create a task to go through the entire test and format all of these lines.